charts/vector/values.yaml

```diff
@@ -169,75 +172,75 @@ resources: {}
   #   cpu: 200m
   #   memory: 256Mi
 
-# lifecycle -- Set vector lifecycle hooks
+# lifecycle -- Set lifecycle hooks for Vector containers.
 lifecycle: {}
   # preStop:
   #   exec:
   #     command:
   #     - /bin/sleep
   #     - "10"
 
-# updateStrategy -- Customize the updateStrategy used to replace Vector Pods
-## Also used for the DeploymentStrategy for Stateless-Aggregators
-## Valid options are used depending on the chosen role
-## Agent (DaemonSetUpdateStrategy): https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)
-## Aggregator (StatefulSetUpdateStrategy): https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/stateful-set-v1/#StatefulSetSpec
-## Stateless-Aggregator (DeploymentStrategy): https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/
+# updateStrategy -- Customize the updateStrategy used to replace Vector Pods, this is also used for the
+# DeploymentStrategy for the "Stateless-Aggregators". Valid options depend on the chosen role.
+
+# Agent (DaemonSetUpdateStrategy): https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)
+# Aggregator (StatefulSetUpdateStrategy): https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/stateful-set-v1/#StatefulSetSpec
+# Stateless-Aggregator (DeploymentStrategy): https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/
 updateStrategy: {}
 #   type: RollingUpdate
 #   rollingUpdate:
 #     maxUnavailable: 1
 
-# terminationGracePeriodSeconds -- Override Vector's terminationGracePeriodSeconds
+# terminationGracePeriodSeconds -- Override Vector's terminationGracePeriodSeconds.
 terminationGracePeriodSeconds: 60
 
-# nodeSelector -- Allow Vector to be scheduled on selected nodes
-## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
-## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+# nodeSelector -- Configure a [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
+# for Vector Pods.
 nodeSelector: {}
 
-# tolerations -- Allow Vector to schedule on tainted nodes
+# tolerations -- Configure Vector Pods to be scheduled on [tainted](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+# nodes.
 tolerations: []
 
-# affinity -- Allow Vector to schedule using affinity rules
-## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+# affinity -- Configure [affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
+# rules for Vector Pods.
 affinity: {}
 
-## Configuration for Vector's Service
+# Configuration for Vector's Service.
 service:
-  # service.enabled -- If true, create and use a Service resource
+  # service.enabled -- If true, create and use a Service resource.
```

Comment:
Should this also say "... a Service resource for Vector." 

, to match the `serviceHEadless.enabled` comment ?

Reply:
Changed in c725ba6